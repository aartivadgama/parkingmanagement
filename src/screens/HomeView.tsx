import React from "react";
import {View, StyleSheet} from "react-native"
import HomeController from "./HomeController";

export default class testView extends HomeController {
    render() {
        return(
            <View style={styles.view}>
            </View>
        );
    }
}

const styles = StyleSheet.create({
    view: {
        flex: 1,
        alignItems: "center",
        backgroundColor: "red",
      },
      separator: {
        height: 20,
      },
});