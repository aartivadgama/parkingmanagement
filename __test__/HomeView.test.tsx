import React from 'react';
import renderer from 'react-test-renderer';
declare module 'react-test-renderer';

import HomeView from '../src/screens/HomeView';

describe('<App />', () => {
  it('has 1 child', () => {
    const tree = renderer.create(<HomeView />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
