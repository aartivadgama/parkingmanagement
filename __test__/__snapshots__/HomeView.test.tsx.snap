// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> has 1 child 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "red",
      "flex": 1,
    }
  }
/>
`;
